src/main/java/gift/DTO/ProductDto.java

```diff
@@ -23,20 +23,24 @@ public class ProductDto {
   @NotBlank
   private String imageUrl;
 
+  private CategoryDto categoryDto;
+
   public ProductDto() {
   }
 
-  public ProductDto(Long id, String name, int price, String imageUrl) {
+  public ProductDto(Long id, String name, int price, String imageUrl, CategoryDto categoryDto) {
     this.id = id;
     this.name = name;
     this.price = price;
     this.imageUrl = imageUrl;
+    this.categoryDto=categoryDto;
   }
 
-  public ProductDto(String name, int price, String imageUrl) {
+  public ProductDto(String name, int price, String imageUrl, CategoryDto categoryDto) {
     this.name = name;
     this.price = price;
     this.imageUrl = imageUrl;
+    this.categoryDto=categoryDto;
```

Comment:
```suggestion
    this.categoryDto = categoryDto;
```
이런부분을 신경쓰시면 좋을거 같아요.

그러나 신경쓰기 어려우니.

https://dev-jhl.tistory.com/entry/IntelliJ-IDEA-%EC%A0%80%EC%9E%A5-%EC%8B%9C-%EC%95%A1%EC%85%98-Actions-on-Save-%EA%B8%B0%EB%8A%A5-%EC%B6%94%EA%B0%80

위의 링크를 참고하여 save action 기능을 쓰시면 좋을거 같네요~

Reply:
알려주셔서 감사합니다!